swagger-schema-official/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "noEmit": true,
```

Comment:
Is this the standard `tsconfig.json`? If you run this without `"types": []`, then the entirety of DefinitelyTyped ends up getting loaded up.